src/main/include/subsystems/Elevator.h

```diff
@@ -26,38 +28,31 @@ namespace t34
 
     Elevator();
 
-    frc2::CommandPtr ElevateToCommand(units::inch_t height);
-    frc2::CommandPtr MoveAlgaeWristToCommand(units::degree_t angle);
-    frc2::CommandPtr MoveCoralWristToCommand(units::degree_t angle);
+    //frc2::CommandPtr ElevateToCommand(units::inch_t height);
+    void ElevateTo(units::inch_t height);
+    void Stop();
+
+    inline bool EndCondition() { return m_elevator_motors_pid.AtSetpoint(); };
 
-    frc2::CommandPtr MoveToLevelCommand(int level);
-    frc2::CommandPtr MoveUpOnceCommand();
-    frc2::CommandPtr MoveDownOnceCommand();
-    frc2::CommandPtr MoveToRestCommand();
+    //frc2::CommandPtr MoveToRestCommand();
 
     frc2::CommandPtr MoveElevatorByPowerCommand(double val);
-    frc2::CommandPtr MoveAlgaeWristByPowerCommand(double val);
-    frc2::CommandPtr MoveCoralWristByPowerCommand(double val);
 
-  private:
-    int m_level;
+    inline units::inch_t GetInitHeight() { return m_init_height; }    
 
-    const units::inch_t m_init_height;
+    inline double GetEncoderUnits() { return m_right_motor.GetSelectedSensorPosition(); }
 
-    const units::degree_t m_init_algae_angle;
-    const units::degree_t m_init_coral_angle;
+    void Periodic() override;
 
-    TalonFX m_right_algae_wrist_motor;
-    TalonFX m_left_algae_wrist_motor;
+  private:
+    int m_level;
 
-    TalonFX m_left_motor;
-    TalonFX m_right_motor;
+    const units::inch_t m_init_height;
 
-    SparkMax m_coral_wrist_motor;
+    TalonSRX m_left_motor;
+    TalonSRX m_right_motor;
 
     frc::PIDController m_elevator_motors_pid;
```

Comment:
Now that we have removed other motors from `Elevator`, we can remove the `elevator` designator from internal members:

```suggestion
    frc::PIDController m_motors_pid;
```

In fact, because we have no other PID controller, perhaps even the `motors` designator is no longer necessary:

```suggestion
    frc::PIDController m_pid;
```